RevenueCatUI/CustomerCenter/View+PresentCustomerCenter.swift

```diff
@@ -278,6 +283,9 @@ private struct PresentingCustomerCenterModifier: ViewModifier {
             .onCustomerCenterCustomActionSelected { actionIdentifier, purchaseIdentifier in
                 onCustomAction?(actionIdentifier, purchaseIdentifier)
             }
+            .onCustomerCenterChangePlansSelected { [changePlansSelected] optionId in
```

Comment:
Why do we need to capture `changePlansSelected`? Note that in `onCustomerCenterManagementOptionSelected`, `managementOptionSelected` is not captured

Reply:
You are right, it's not needed